.github/workflows/build.yaml

```diff
@@ -0,0 +1,45 @@
+name: build
+on:
+  push: {}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Unshallow
+        run: git fetch --prune --unshallow
```

Comment:
What's this do here? It might help to mention that this is a Git operation in the name field.

Reply:
not entirely sure. its just part o the goreleaser template